vscode/package.json

```
@@ -797,7 +785,10 @@
           "default": {
             "*": true,
             "scminput": false
-          }
+          },
+          "examples": [
+            "\"JavaScript\": true"
```

```suggestion
            "\"javascript\": true"
```

This needs to be lowercased 🙂